src/main/java/net/minecraftforge/common/ForgeConfig.java

```diff
@@ -52,62 +52,62 @@
 
         Server(ForgeConfigSpec.Builder builder) {
             builder.comment("Server configuration settings")
-                   .push("server");
```

Comment:
Please leave all the indentation as-is, it just bloats the already huge diff

Reply:
Heh, you should see my old PR